monkey/infection_monkey/utils/capture_output.py

```diff
@@ -7,7 +7,6 @@ def __enter__(self) -> None:
         self._orig_stdout = sys.stdout
         self._new_stdout = io.StringIO()
         sys.stdout = self._new_stdout
-        return self
```

Comment:
I believe that the `__enter__()` method on a context manger needs to return something. I think the typehint is wrong. You'll have to `from __future__ import annotations` in order for the typehint to be able to reference the class IIRC.

Reply:
It looks like you do if the user needs to refer to the object using `as`, which we do. I'll update it